Spigot-Server-Patches/0733-Add-vs-code-files-to-.gitignore.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
Stuff like this should be in an earlier patch, maybe the pom changes patch unless there was a more misc one